### ISSUE TITLE
Renaming a namespace export's name does not update name in import specifiers

### ISSUE THREAD

**Issue by dsherret** [microsoft/TypeScript#36836]
**TypeScript Version:**  3.9.0-dev20200215

**Search Terms:** Namespace export rename

**Code**

*a.ts*:

```ts
export class Test {
    name = 5;
}
```

*b.ts*:

```ts
export * as ns from "./a"; // <-- rename ns to nsTest
```

*c.ts*:

```ts
import { ns } from "./b";

const t = new ns.Test();
```

**Expected behavior:**

*b.ts*:

```ts
export * as nsTest from "./a";
```

*c.ts* (depends on setting "Rename Shorthand Properties" I think):

```ts
import { nsTest } from "./b";

const t = new nsTest.Test();
```

**Actual behavior:**

*b.ts*:

```ts
export * as nsTest from "./a";
```

*c.ts*:

```ts
import { ns } from "./b";

const t = new ns.Test();
```

So, *c.ts* does not have any of its identifiers renamed.

![2020-02-17_11-19-06](https://user-images.githubusercontent.com/1609021/74670650-9fab8e80-5177-11ea-98c8-01b8c6801c2d.gif)

**Comment by dsherret** [comment]
Doesn't seem to work in this scenario either:

```ts
import("./b").then(test => {
    new test.ns.Test();
});
```

**Comment by jessetrinity** [comment]
Fixed in #39533
